tests/losses/test_unified_focal_loss.py

```
@@ -19,47 +19,86 @@
-        loss = AsymmetricUnifiedFocalLoss()
-        result = loss(**input_data)
-        np.testing.assert_allclose(result.detach().cpu().numpy(), expected_val, atol=1e-4, rtol=1e-4)
+    @parameterized.expand([TEST_CASE_BINARY_LOGITS, TEST_CASE_BINARY_2CH, TEST_CASE_MULTICLASS_PERFECT])
+    def test_perfect_prediction(self, input_data, expected_val, args):
+        loss_func = AsymmetricUnifiedFocalLoss(**args)
+        result = loss_func(**input_data)
+        # Using a relaxed tolerance for logits -> probability conversion
+        np.testing.assert_allclose(result.detach().cpu().numpy(), expected_val, atol=1e-3, rtol=1e-3)
+
+    @parameterized.expand([TEST_CASE_MULTICLASS_WRONG])
+    def test_wrong_prediction(self, input_data, expected_val, args):
```

Ruff (0.14.10)

Unused method argument: `expected_val`

(ARG002)